lib/generators/phlex/install/install_generator.rb

```diff
@@ -6,36 +6,24 @@ class InstallGenerator < ::Rails::Generators::Base
 
 		APPLICATION_CONFIGURATION_PATH = Rails.root.join("config/application.rb")
 		TAILWIND_CONFIGURATION_PATH = Rails.root.join("tailwind.config.js")
-
-		def autoload_components
-			return unless APPLICATION_CONFIGURATION_PATH.exist?
-
-			inject_into_class(
-				APPLICATION_CONFIGURATION_PATH,
-				"Application",
-				%(    config.autoload_paths << "\#{root}/app/views/components"\n)
-			)
-		end
-
-		def autoload_layouts
-			return unless APPLICATION_CONFIGURATION_PATH.exist?
-
-			inject_into_class(
-				APPLICATION_CONFIGURATION_PATH,
-				"Application",
-				%(    config.autoload_paths << "\#{root}/app/views/layouts"\n)
-			)
-		end
-
-		def autoload_views
-			return unless APPLICATION_CONFIGURATION_PATH.exist?
-
-			inject_into_class(
-				APPLICATION_CONFIGURATION_PATH,
-				"Application",
-				%(    config.autoload_paths << "\#{root}/app/views"\n)
-			)
-		end
+    ADD_EXTRA_AUTOLOAD_PATHS_CODE = <<-ADD_EXTRA_AUTOLOAD_PATHS_CODE
+    config.autoload_paths.push(
+      "\#{root}/app/views/components",
+      "\#{root}/app/views",
+      "\#{root}/app/views/layouts"
+    )
+
+    ADD_EXTRA_AUTOLOAD_PATHS_CODE
```

Comment:
I don’t think this line does anything, seems to be leftover from debugging.

Reply:
I think it does: closes the heredoc from line 9 ;)

Probably three occurrences of ADD_EXTRA_AUTOLOAD_PATHS_CODE and the extra empty line (on line 15) was a bit disorienting. 

Not sure what's the deal with using the same string 3x (yay/nay/what's the best practice).

As for the extra line. I prefer the extra line - it's the difference between 

```ruby
    config.autoload_paths.push(
      "#{root}/app/views/components",
      "#{root}/app/views",
      "#{root}/app/views/layouts"
    )
    config.application_name = "railsnew.io" # look ma, no empty line above me
```
and
 
```ruby
    config.autoload_paths.push(
      "#{root}/app/views/components",
      "#{root}/app/views",
      "#{root}/app/views/layouts"
    )
    
    config.application_name = "railsnew.io" # 👌👌👌
```    
    
While I prefer the second version (I'm a sucker for little things like this), in this case I don't mind either way as `application.rb` is not a file anyone is frequently working with.

Reply:
> I think it does: closes the heredoc from line 9 ;)

🤦 Oh of course! I thought it was a void call to the constant.


The extra line makes sense too.